### ISSUE TITLE
.onTapGesture missing

### ISSUE THREAD

**Issue by Fahrenberg** [diniska/modal-view#11]
Is it possible to add a .onTapGesture to ModalLink/ModalPresenter?

**Comment by diniska** [comment]
Yes, you can add `onTapGesture` to any view according to SwiftUI docs. Here is the screenshot of the method definition. I don't think that we need to do something specific here unless it works in an unexpected way for you.
![Снимок экрана 2020-10-16 в 23 54 21](https://user-images.githubusercontent.com/2674368/96260565-eb664e80-100a-11eb-8951-b7aa6b1619e8.png)
